.vscode/settings.json

```diff
@@ -21,5 +21,6 @@
   "peacock.color": "#05822b",
   "cSpell.words": [
     "lusitana"
-  ]
+  ],
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
This workspace setting fix a TS lint issue that is existing if you are using an old version of TS which is the default of the VSCode